src/pages/resultsView/ResultsViewURLWrapper.ts

```diff
@@ -67,9 +84,26 @@ export default class ResultsViewURLWrapper extends URLWrapper<
                 treatment_list: { isSessionProp: false },
                 exclude_germline_mutations: { isSessionProp: false },
                 patient_enrichments: { isSessionProp: false },
+
                 comparison_subtab: { isSessionProp: false },
                 comparison_overlapStrategy: { isSessionProp: false },
 
+                plots_selectedGeneOption_horz: { isSessionProp: false },
```

Comment:
should we consider keeping these as one object that gets serialized to URL param?   we could probably even adapt urlWrapper to intercept change signaling in the same way as we do at the top level.

Reply:
A couple of cons to that that are popping up:
- typescript won't be able to check the specific props within the serialized object
- the url will get even uglier

What makes you want to put them into one object?

Reply:
Reasons:

1. I can foresee a use case where we want to be able to clear these things as a group in some cases, for example when a query mod renders the configuration invalid. (We got lucky in that oncoprint can handle that circumstance gracefully.) 
2. The fact that prefix these props shows that they are indeed a group. Object is a nicer way of grouping.   If the only drawback is ugliness, i think that we should stop caring about ugliness.  The URL is so long that it's not really composable or readable by a human being anyway.  

Typing
I don't see why we can't type them  as a node of the parent object 
```
plots_config: { ... props, isSessionProp:bool }